tests/test_cli.py

```diff
@@ -116,6 +116,22 @@ def test_render_output_build_path(testing_workdir, testing_metadata, capfd, capl
     assert error == ""
 
 
+def test_render_output_build_path_and_file(testing_workdir, testing_metadata, capfd, caplog):
+    api.output_yaml(testing_metadata, 'meta.yaml')
+    
+    with TemporaryDirectory() as tmpdir:
```

Comment:
Instead of using ``TemporaryDirectory`` you can just use a ``pytest`` fixture called ``tmpdir``

Reply:
the ``pytest`` fixture ``tmpdir`` is a ``py.path .local``
Please see
http://doc.pytest.org/en/latest/tmpdir.html#the-tmpdir-fixture

Reply:
Probably it will simplify a bit the test :)
If you need any help please ping me!